terraform/modules/function/main.tf

```diff
@@ -145,6 +145,9 @@ module "zip_bucket" {
     "arn:aws:iam::${data.aws_ssm_parameter.prod_account[0].value}:role/delegatedadmin/developer/${var.app}-prod-github-actions",
     "arn:aws:iam::${data.aws_ssm_parameter.sbx_account[0].value}:role/delegatedadmin/developer/${var.app}-sbx-github-actions",
   ] : []
+  
+  app = var.app
+  env = var.env
```

Comment:
These will also need to be added here: https://github.com/CMSgov/ab2d-bcda-dpc-platform/blob/6cc555cad64f7cb69d2a417f13d34657316b9c1a/terraform/services/tfstate/main.tf#L7

The tfstate workflows should also be updated to include bucket and table module paths.